plugins/module_utils/dnac.py

```diff
@@ -1014,8 +1014,11 @@ def validate_int(item, param_spec, param_name, invalid_params):
             "range_max": 100    # Optional: maximum allowed value
         }
     """
```

Comment:
Can we rename the API to  'validate_integer_within_range' or 'validate_integer_constraints'